code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -89,7 +89,9 @@
 	/obj/item/clothing/head/sombrero,
 	/obj/item/clothing/head/witchunter_hat)
 
-
+	can_buckle = TRUE
+	buckle_lying = FALSE
+	var/datum/riding/cyborg/riding_datum = null
```

Comment:
if you're gonna do this wouldnt it be better to just put the var declaration in atom